### PR TITLE
Add identitywallet function for getting tx status

### PR DESCRIPTION
**Pull request by cducrest**
requires: https://github.com/trustlines-protocol/relay/pull/426

### CODE REVIEW

File: src/providers/RelayProvider.ts [hunk 1]
```diff
@@ -133,6 +137,16 @@ export class RelayProvider implements TLProvider {
     }
   }
 
+  public async getMetaTxStatus(
+    identityAddress: string,
+    metaTransactionHash: string
+  ) {
+    const { MetaTransactionStatus } = await this.fetchEndpoint<any>(
+      `/identities/${identityAddress}/meta-transactions/${metaTransactionHash}`
```
Comment (berndbohmeier) [inline src/providers/RelayProvider.ts]: here you used `meta-transactions` but did you not use `meta-transaction-status` in the relay. My suggestion would be to use the link here.

File: src/providers/RelayProvider.ts [hunk 2]
```diff
@@ -133,6 +137,16 @@ export class RelayProvider implements TLProvider {
     }
   }
 
+  public async getMetaTxStatus(
+    identityAddress: string,
+    metaTransactionHash: string
+  ) {
+    const { MetaTransactionStatus } = await this.fetchEndpoint<any>(
```
Comment (berndbohmeier) [inline src/providers/RelayProvider.ts]: I would return the whole object here so that we can later use also different values returned by the relay.

**Comment by cducrest** [comment]
This should be mergeable but it requires new version of the relay so I'll wait for it

**Comment by cducrest** [comment]
Merged as part of other PR

**Pull request closed without merging**